<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::CertificateDialog</class>
 <widget class="QDialog" name="Otter::CertificateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2,0,3,0,1,0">
   <item>
    <widget class="QLabel" name="chainLabel">
     <property name="text">
      <string>Certificate chain:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Otter::ItemViewWidget" name="chainItemView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="fieldsLabel">
     <property name="text">
      <string>Certificate fields:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Otter::ItemViewWidget" name="detailsItemView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>Field value:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Otter::TextEditWidget" name="valueTextEditWidget">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::ItemViewWidget</class>
   <extends>QTreeView</extends>
   <header>src/ui/ItemViewWidget.h</header>
  </customwidget>
  <customwidget>
   <class>Otter::TextEditWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>src/ui/TextEditWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Otter::CertificateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
